index.html

```diff
@@ -174,26 +207,25 @@ <h2 class="attribute-name">Skills</h2>
                         <li>API Design</li>
 					</ul>
                 </div>
-            </li>
-        </ul>
     </div>
 </section>
 <section class="contact-section">
+    <div class="contact-wrapper">
     <div class="contact-header">
```

Comment:
- [ ] please kindly work on the display of your contact me section on the desktop version as it is currently inconsistent with the figma template as shown below. it is too close to the borders to the left and to the right. rather than using fixed margin i would suggest using a fixed width preferably in `%` and centering it in the middle using `margin:auto` :+1: 

- figma design
![cs-f](https://user-images.githubusercontent.com/89055502/236343034-9544d819-631f-4daf-8cfd-d9feeb4d1062.png)
- your design
![cs-s](https://user-images.githubusercontent.com/89055502/236343058-c05c9d88-c689-4b0c-8e49-1db2af32ebaf.png)